plenary/plenary-13.yaml

```diff
@@ -0,0 +1,238 @@
+        Thereafter ISO/TC 154 secretariat will submit do. N 227 rev. 1 to ISO/TMB for approval.
+
+
+  - dates:
+      - 1997-06-12
+      - 1997-06-13
+    identifier: 127
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: requests
+        date_effective:
+          start: 1997-06-12
+          end: 1997-06-13
+        message: ISO/TC 154 request its secretariat to submit the procedural comments of the member body of Canada on ISO DIS 9735 Part 1, 2, 3 and 8 to ISO/CS for clarification
+
+
+  - dates:
+      - 1997-06-12
+      - 1997-06-13
+    identifier: 128
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: resolves
+        date_effective:
+          start: 1997-06-12
+          end: 1997-06-13
+        message: |
+        ISO/TC 154 resolves to establish an editing committee for the revision of ISO 9735, to request Don Trafford to serve as chair of this editing committee and Bernd Boesler as its secretary.
+        The chair of the editing committee shall in consulation with his secretary propose the core membership and set a date for a first meeting of the editing committee respecting the time consideration as specified in the ISO Directives.
+
+
+  - dates:
+      - 1997-06-12
+      - 1997-06-13
+    identifier: 129
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: recommends
+        date_effective:
+          start: 1997-06-12
+          end: 1997-06-13
+        message: ISO/TC 154 recommends that the member bodies of ISO/TC 154 which have submitted substantial technical comments on Parts 1, 2, 3 and 8 of ISO DIS 9735 should be invited to assist the editing committee meeting, though their non-attendance shall not cause postponement of the meeting
+
+
+  - dates:
+      - 1997-06-12
+      - 1997-06-13
+    identifier: 130
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: recognises
+        date_effective:
+          start: 1997-06-12
+          end: 1997-06-13
+        message: ISO/TC 154 recognises and approves the nomination of Don Trafford as chair of the Joint Syntax Development Working Group (JSDWG) and appoints Bernd Boesler as secretary
+
+
+  - dates:
+      - 1997-06-12
+      - 1997-06-13
+    identifier: 131
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: suppports
+        date_effective:
+          start: 1997-06-12
+          end: 1997-06-13
+        message: ISO/TC 154 supports Res. 55/1997 on the submission of the UN/EDIFACT Directives from the TMB and requests that it be considered in conjunction with the recommendations given in the High Level Steering Group (HLSG) on CALS report to the TMB (1997-05-25)
```

Comment:
In this resolution, I used the word `supports` as an action type. 
(I've noticed the typo `suppports` and corrected it)

Reply:
Maybe this works better.

```yaml
    actions:
      - type: supports
        date_effective:
          start: 1997-06-12
          end: 1997-06-13
        message: ISO/TC 154 supports Res. 55/1997 on the submission of the UN/EDIFACT Directives from the TMB
      - type: requests
        date_effective:
          start: 1997-06-12
          end: 1997-06-13        
        message: requests that Res. 55/1997 be considered in conjunction with the recommendations given in the High Level Steering Group (HLSG) on CALS report to the TMB (1997-05-25)

    message: |
      ISO/TC 154 supports Res. 55/1997 on the submission of the UN/EDIFACT Directives from the TMB and requests that it be considered in conjunction with the recommendations given in the High Level Steering Group (HLSG) on CALS report to the TMB (1997-05-25)
```